### PR TITLE
Updating all Python files to have ASF licenses and pass PEP8 tests

### PR DESCRIPTION
**Pull request by mrutkows**
Required for Apache and to pass new scanCode.py tests going forward.

### CODE REVIEW

File: provider/app.py [hunk 1]
```diff
@@ -154,15 +173,17 @@ def healthRoute():
...
 
 
 def createAndRunConsumer(triggerFQN, params, record=True):
-    if app.config['TESTING'] == True:
+    if app.config['TESTING'] is True:
         logging.debug("Just testing")
     else:
         # generate a random uuid for new triggers
-        if not 'uuid' in params:
+        if 'uuid' not in params:
```
Comment (bjustin-ibm) [inline provider/app.py]: Is this change necessary? This change is grammatically less readable.

File: provider/app.py [hunk 2]
```diff
@@ -205,7 +226,8 @@ def main():
 
     # Make sure we log to the console
     streamHandler = logging.StreamHandler()
-    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] [??] [kafkatriggers] %(message)s')
+    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] '
```
Comment (bjustin-ibm) [inline provider/app.py]: Are all these formatting-only changes necessary? If not, I would suggest removing them since they are just muddying the history of this file. If they are necessary, then what is to prevent us from creating more formatting issues in the future?

File: provider/consumer.py [hunk 3]
```diff
@@ -65,20 +72,25 @@ def start(self):
...
         with self.__lock:
             self.__restartCount += 1
 
-        logging.info('[{}] Quietly shutting down consumer for restart'.format(self.trigger))
+        logging.info("[{}] Quietly shutting down consumer for "
+                     "restart".format(self.trigger))
         self.thread.setDesiredState(Consumer.State.Restart)
         self.thread.join()
-        logging.info('Consumer has shut down')
+        logging.info("Consumer has shut down")
```
Comment (bjustin-ibm) [inline provider/consumer.py]: hmmm... quote usage throughout these changes is inconsistent. What is the reason for changing some but not others?

File: provider/consumer.py [hunk 4]
```diff
@@ -201,58 +219,71 @@ def run(self):
                 if len(messages) > 0:
                     self.__fireTrigger(messages)
 
-            logging.info("[{}] Consumer exiting main loop".format(self.trigger))
+            logging.info("[{}] Consumer exiting main "
```
Comment (bjustin-ibm) [inline provider/consumer.py]: What is the reason for breaking this string?

**Comment by mrutkows** [comment]
@bjustin These are all changes to adhere to PEP8 (<80 char lines and other things) that may appear "cosmetic" but to the Python community makes a significant different in terms of "correctness" to Python agreed-upon standards.

File: provider/app.py (hunk 1 above)
Comment (mrutkows) [inline provider/app.py]: Yes, it is more correct to test truth (boolean) in this manner in Python.

File: provider/app.py (hunk 2 above)
Comment (mrutkows) [inline provider/app.py]: @bjustin-ibm we are adding python PEP8 linting for all python files over time; Adhering to the <80 char line limit will be mandatory.

File: provider/consumer.py (hunk 3 above)
Comment (mrutkows) [inline provider/consumer.py]: @bjustin-ibm  Single quotes for symbolic strings, double quotes for (potentially) interpolated (or language translatable) strings.

**Comment by bjustin-ibm** [comment]
> These are all changes to adhere to PEP8 (<80 char lines and other things) that may appear "cosmetic" but to the Python community makes a significant different in terms of "correctness" to Python agreed-upon standards.

I respectfully disagree about the line length. A quick survey of Stackoverflow shows that the 79 character limit imposed by PEP8 is extremely contentious and hotly debated. 

- https://stackoverflow.com/a/17319775
- https://stackoverflow.com/a/2793525

If you read PEP8 itself, they not only recognize this contention, but also explain that it is done only for readability on legacy displays which are only capable of displaying content 80 characters wide. It also allows for development teams to agree to enforce a longer line length: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

As a development team, I think we can agree that we are not developing this code on legacy displays that are incapable of displaying more than 80 characters and _also_ incapable of soft-wrapping long lines. I contend that the string breaking changes included in this PR actually reduce the readability of the code, rather than enhance it - e.g. [this proposed change](https://github.com/apache/incubator-openwhisk-package-kafka/pull/149/files/d91656bd7a3e5f68384489e998132efc99fe7966#diff-c7f0a1458defdca4247d971225ee27baR131).

I welcome a standard code formatting so long as it can be automated, and does not detract from the actual readability of the code by imposing an arcane limitation. 

I have some concrete suggestions for how to proceed:
- Please include a `pep8.rc` file that matches what scanCode expects. This will give us awareness of what needs to change without waiting for Travis, as well as the ability to configure our development tools to automatically fix the style issues.
- In that config file, let's set the line limit to either 120 or 160 characters. This will bring some amount of rigor and conformity to our formatting, without necessarily decreasing the readability.

**Comment by rabbah** [comment]
+1 - we discussed this on the other repo and decided to not stick to the 80 chars per line. It really makes the code unreadable and as Justin pointed out these are plenty of reasons not to do this.

File: provider/app.py (hunk 2 above)
Comment (rabbah) [inline provider/app.py]: let's back off "mandatory" - we relaxed this 80 char limit in the main repo iirc.

**Comment by mrutkows** [comment]
@bjustin-ibm will resubmit without any of the PEP8 style guide line length changes.
